server/plugin.go

```diff
@@ -551,8 +551,14 @@ func addFields(fields []*model.SlackAttachmentField, title, msg string, short bo
 func messageToJSON(message string) ([]byte, error) {
 	messagefields := strings.Split(message, "\n")
 	if len(messagefields) > 0 {
-		// examine if the message refers to a cloudformation event by checking if StackId field is included in the first line
-		if strings.Split(messagefields[0], "=")[0] != "StackId" {
+		// examine if the message refers to a cloudformation event by checking if a valid StackId field is included in the first line
+		stackIdParts := strings.Split(messagefields[0], "=")
+		if len(stackIdParts) == 2 && stackIdParts[0] == "StackId" {
```

Comment:
How confident are we that a CloudFormation event will always have `StackId` on its first line?

Reply:
When testing with stacks where each one had different resources, the order of the StackId field in the Cloudformation message was the same between different events.